spec.html

```diff
@@ -25877,6 +26626,144 @@ <h1>Properties of Number Instances</h1>
+        <emu-alg>
+          1. Assert: Type(_number_) is Number.
+          1. If IsInteger(_number_) is *false*, throw a *RangeError* exception.
+          1. Return a BigInt representing the mathematical value of _number_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-bigint-constructor-number-value">
+        <h1>BigInt ( _value_ )</h1>
+        <p>When `BigInt` is called with argument _value_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is not *undefined*, throw a *TypeError* exception.
+          1. Let _prim_ be ? ToPrimitive(_value_, hint Number).
+          1. If Type(_prim_) is Number, return ? NumberToBigInt(_prim_).
+          1. Otherwise, return ? ToBigInt(_value_).
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-bigint-constructor">
+      <h1>Properties of the BigInt Constructor</h1>
+      <p>The value of the [[Prototype]] internal slot of the BigInt constructor is the intrinsic object %FunctionPrototype%.</p>
+      <p>The BigInt constructor has the following properties:</p>
+
+      <!-- es6num="20.1.2.2" -->
+      <emu-clause id="sec-bigint.asuintn">
+        <h1>BigInt.asUintN ( _bits_, _bigint_ )</h1>
+        <p>When the `BigInt.asUintN` function is called with two arguments _bits_ and _bigint_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _bits_ be ? ToIndex(_bits_).
+          1. Let _bigint_ be ? ToBigInt(_bigint_).
+          1. Return a BigInt representing _bigint_ modulo 2<sup>_bits_</sup>.
+        </emu-alg>
+      </emu-clause>
+
+      <!-- es6num="20.1.2.3" -->
+      <emu-clause id="sec-bigint.asintn">
+        <h1>BigInt.asIntN ( _bits_, _bigint_ )</h1>
+        <p>When the `BigInt.asIntN` is called with two arguments _bits_ and _bigint_, the following steps are taken:</p>
+        <emu-alg>
+          1. Let _bits_ be ? ToIndex(_bits_).
+          1. Let _bigint_ be ? ToBigInt(_bigint_).
+          1. Let _mod_ be a BigInt representing _bigint_ modulo 2<sup>_bits_</sup>.
+          1. If _mod_ &ge; 2<sup>_bits_ - 1</sup>, return _mod_ - 2<sup>_bits_</sup>; otherwise, return _mod_.
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-bigint.prototype">
+        <h1>BigInt.prototype</h1>
+        <p>The initial value of `BigInt.prototype` is the intrinsic object %BigIntPrototype%.</p>
+        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-bigint-prototype-object">
+      <h1>Properties of the BigInt Prototype Object</h1>
+      <p>The BigInt prototype object is the intrinsic object <dfn>%BigIntPrototype%</dfn>. The BigInt prototype object is an ordinary object. The BigInt prototype is not a BigInt object; it does not have a [[BigIntData]] internal slot.</p>
```

Comment:
Other parts of the specification have moved to a bullet-point format, e.g., see https://tc39.github.io/ecma262/#sec-properties-of-the-number-prototype-object . We should follow that for BigInt as well.